test/over_react/util/class_names_test.dart

```diff
@@ -239,6 +239,53 @@ main() {
             }));
           });
         });
+
+        group('merge', () {
+          test('merges the className from the provided builder instance', () {
+            var otherBuilder = new ClassNameBuilder();
+
+            builder.add('a');
+            otherBuilder.add('b');
+  
+            builder.merge(otherBuilder);
+            expect(builder.toClassName(), equals('a b'));
```

Comment:
@kealjones-wk you don't have to change this, but `equals` is the default `Matcher` for the second arg of `expect`, so it can be omitted.  Like I said - you definitely do not have to change this, as we are certainly not consistent with this in any of our repos... but I thought I'd point it out.

```dart
expect(builder.toClassName(), 'a b');
```